includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -364,7 +407,7 @@ protected function prepare_line_items( $order ) {
 	protected function add_line_item( $item_name, $quantity = 1, $amount = 0.0, $item_number = '' ) {
 		$index = ( count( $this->line_items ) / 4 ) + 1;
 
-		if ( $amount < 0 || $index > 9 ) {
+		if ( $amount < 0 ) {
```

Comment:
Someone mentioned a 'max' amount in one of the issues too - shall we take care of that now also?

Reply:
If it cannot be taken care of easily, don't worry. It is edge case.

Reply:
I don't know if there is a good limit to use. The [wording on Paypal website](https://www.paypal.com/us/selfhelp/article/what-is-the-maximum-amount-i-can-send-with-my-paypal-account-faq732#personal) is a bit vague:
> If your account is Verified, there’s no limit on the total amount of money you can send from your account. You can send up to $60,000, but may be limited to $10,000, in a single transaction. (This amount can vary, though, depending on your currency.)

> If your account is Unverified, there’s a limit on the total amount of money you can send from your account.

Do you think we should add a limit of $10,000 based on this?

I actually think it's perhaps the Paypal's responsibility to tell you if you have sufficient funds/how much you can pay, so I don't think it belongs to WooCommerce core? That's a bit different from invalid request which can be too long or has too many fields.

Reply:
Lets leave this one then.